astropy/coordinates/tests/test_frames.py

```diff
@@ -1437,6 +1437,25 @@ def test_galactocentric_defaults(reset_galactocentric_defaults):
         else:
             assert getattr(galcen_40, k) == getattr(galcen_latest, k)
 
+    # test validate Galactocentric
+    with galactocentric_frame_defaults.set('latest'):
+        params = galactocentric_frame_defaults.validate(galcen_latest)
+        references = galcen_latest.frame_attribute_references
+
+        assert galactocentric_frame_defaults._value == params
```

Comment:
Total detail, but I'd test the public attributes.

Reply:
- [ ] Done